bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -107,4 +107,9 @@ impl WasmBezier {
 		let local_extrema = self.0.local_extrema();
 		JsValue::from_serde(&serde_json::to_string(&local_extrema).unwrap()).unwrap()
 	}
+
+	pub fn inflections(&self) -> JsValue {
+		let inflections = self.0.inflections();
+		JsValue::from_serde(&serde_json::to_string(&inflections).unwrap()).unwrap()
```

Comment:
Could we create a template function that does the `serde` / `json` wrapping and then use it both here and in extrema?